pkg/logger/logger.go

```diff
@@ -16,16 +16,23 @@ const (
 )
 
 var (
-	l    logr.Logger = logr.Discard()
-	once sync.Once
+	l              logr.Logger = logr.Discard()
+	once           sync.Once
+	outputFilename string
```

Comment:
We should think about grouping these 3 (or at least the logger and output file) in a struct
Can be a follow up

Reply:
Agreed